wolfcrypt/src/asn.c

```diff
@@ -13955,9 +13955,22 @@ static int DecodeAltNames(const byte* input, int sz, DecodedCert* cert)
             dnsEntry->len = strLen;
             XMEMCPY(dnsEntry->name, &input[idx], strLen);
             dnsEntry->name[strLen] = '\0';
+            dnsEntry->next = NULL;
```

Comment:
Can these three new sections be placed into a new function? I see it used similarly in three places. Also there is an extra level of brace.

Also, I am not sure changing this at the asn.c level is a good idea, since it may affect existing users. I would agree with changing it at the openssl compatibility level.